src/services/DataService.ts

```diff
@@ -96,11 +96,13 @@ export async function getServiceRequestSummary(): Promise<ServiceRequestSummary[
     service_category,
     clients(first_name),
     pets(name),
-    team_members(first_name),
-    urgent
+    team_members(first_name, email),
+    urgent,
+    status,
+    modified_at
     `)
-    .order('created_at', { ascending: false })
-    .limit(10);
```

Comment:
I know we talked about how the browser can store a lot more than 10 rows so this is a good change. I wonder though if we should still have an upper limit, like 100,000. For instance, what if there is a bug and millions of rows are added, it could be a major problem for the client